components/sbm-recipes-mule-to-boot/src/test/java/org/springframework/sbm/mule/actions/SubflowsTest.java

```diff
@@ -169,15 +168,15 @@ public void generatedFlowShouldHaveMethodParams() {
                         "@Configuration\n" +
                         "public class FlowConfigurations {\n" +
                         "    @Bean\n" +
-                        "    IntegrationFlow amqp_muleFlow(ConnectionFactory connectionFactory, IntegrationFlow outToAMQP) {\n" +
+                        "    IntegrationFlow amqp_muleFlow(org.springframework.integration.dsl.IntegrationFlow outToAMQP, org.springframework.amqp.rabbit.connection.ConnectionFactory connectionFactory) {\n" +
```

Comment:
What changed that we need to provide fq names here now?

Reply:
We had an issue that if a single file imports two classes which has same name, we were not handling that case. We needed a better management of import than just using simple class name. 

Perhaps an open rewrite recipe could called programatically after  mule to boot recipe ?

Reply:
Let's discuss in person later?